husky_description/package.xml

```diff
@@ -19,14 +19,13 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roslaunch</build_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>urdf</run_depend>
   <run_depend>xacro</run_depend>
   <run_depend>ur_description</run_depend>
   <run_depend>lms1xx</run_depend>
 
-
+  <test_depend>roslaunch</test_depend>
```

Comment:
But it won't work, because `<test_depend>` in format one doesn't pull it in at build time:

http://www.ros.org/reps/rep-0140.html#making-test-depend-easier-to-use